packages/opencensus-web-instrumentation-zone/package.json

```diff
@@ -64,5 +64,5 @@
     "@opencensus/web-instrumentation-zone-peer-dep": "^0.0.3",
     "zone.js": "~0.9.1"
   },
-  "sideEffects": false
+  "sideEffects": true
```

Comment:
Nice catch here! And a good reason for this to be a separate package from the `-peer-dep` one!